schema/snapcraft.json

```diff
@@ -944,6 +944,7 @@
                             "items": {
                                 "type": "string",
                                 "enum": [
+                                    "enable-patchelf",
                                     "no-patchelf",
```

Comment:
I wonder if we should alias no-patchelf to disable-patchelf with a deprecation message when the other is used. That is not on necessarily work to do on this PR

Reply:
Yeah that would seems reasonable.